build.gradle

```diff
@@ -91,7 +91,17 @@ allprojects {
     apply plugin: 'base'
 
     afterEvaluate { project ->
-        composeBuild.dependsOn(project.tasks.assemble)
+        def composeDeps = [
+                "airbyte-config:init",
+                "airbyte-db",
+                "airbyte-scheduler",
+                "airbyte-server",
+                "airbyte-webapp",
+        ].toSet().asImmutable()
+
+        if (project.name in composeDeps) {
+            composeBuild.dependsOn(project.tasks.assemble)
```

Comment:
should i be depending on this? or on something like this? `files(project(project.name).airbyteDocker.outputs`

Reply:
your dependcy is correct right now

Reply:
building the image only depends on the jar. We might actually want to use the docker plugin here instead of docker compose.

Reply:
cool. thanks! going to merge as is and then we can move to the docker plugin after if we'd like.